docs/plugins/bonding.md

```diff
@@ -0,0 +1,66 @@
+(plugin-bonding)=
+# Host-side link bonding
+
+Linux networking has long supported *bonding*, the ability to use multiple links simultaneously. Netlab supports bonding with LACP through the *lag* module,
+this plugin adds support for the other bonding modes (that don't require any special configuration on peers)
+
+```eval_rst
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+   :backlinks: none
+```
+
+## Using the Plugin
+
+* Add `plugin: [ bonding ]` to the lab topology.
+* Include the **bonding.ifindex** attribute in any links that need to be bonded
+
+### Supported attributes
+
+The plugin adds the following attributes:
+* **bonding.mode** (string, one of active-backup, balance-tlb, or balance-alb) -- the bonding mode to use, default `active-backup`
+
+Additional interface level attributes:
+* **bonding.ifindex** (int,mandatory) -- the interface index for the bonding device; links with matching ifindex are bonded together
+* **bonding.primary** (bool) -- optional flag to mark this interface as primary, default *False*
```

Comment:
Document what happens if this is not set on any interface

Reply:
ok